post-to-aws/src/main/java/no/sikt/nva/brage/migration/aws/S3StorageImpl.java

```diff
@@ -185,6 +185,9 @@ private String getResourceDirectory(Record record) {
     }
 
     private String createKey(Record record, String filename) {
+        if (StringUtils.isBlank(record.getBrageLocation())) {
+            return Path.of(customer, record.getCristinId(), filename).toString();
+        }
```

Comment:
This is a "hack" to workaround the xxx/y brageLocation-pattern. If no brageLocation is set, we just use the cristin-id as dir-name

Reply:
Better to set "unis" or something along those lines instead of empty string?